components/board/pi/impl/servo.go

```diff
@@ -63,7 +63,7 @@ func init() {
 						return nil, errors.Errorf("gpioServo failed with %d", setPos)
 					}
 				} else {
-					setPos := C.gpioServo(theServo.pin, C.uint(angleToPulseWidth(int(*attr.StartPos))))
+					setPos := C.gpioServo(theServo.pin, C.uint(angleToPulseWidth(int(*attr.StartPos), int(theServo.max))))
```

Comment:
[minor] Default to 180 since this field is populated by the config and is currently omitted when empty (defaults to zero by golang)